resources/console/profile-production.yaml

```diff
@@ -0,0 +1,14 @@
+web:
+  deployment:
+    resources:
+      requests:
+        memory: 64Mi
+      limits:
+        memory: 128Mi
+backend: 
+  deployment:
+    resources:
+      requests:
+        memory: 64Mi
```

Comment:
^ As in the profile-evaluation case - I think it'll never consume less than ~100Mi.